File: Models/Servicio.cs

namespace Lavanderia.Models
{
public class Servicio
{
    public int Id { get; set; }

    public string Nombre { get; set; } = string.Empty;

    public string Descripcion { get; set; } = string.Empty;

    public decimal Precio { get; set; }

    public int DuracionEstimada { get; set; }

    public bool Activo { get; set; } = true;

    public ICollection<Reserva> Reservas { get; set; } = new List<Reserva>();
}
}

File: src/models/Servicio.js
/**
 * Modelo Servicio
 * Basado en la tabla SQL "Servicios" (Id, Nombre, Descripcion, Precio, DuracionEstimada, Activo).
 */

/**
 * @typedef {Object} Servicio
 * @property {string} id
 * @property {string} nombre
 * @property {string} descripcion
 * @property {number} precio
 * @property {number} duracionEstimada - en minutos
 * @property {boolean} activo
 * @property {string} icono - emoji/ícono representativo del tipo de servicio
 */

export const crearServicio = ({
  id,
  nombre,
  descripcion,
  precio,
  duracionEstimada,
  activo = true,
  icono = '🧺',
}) => ({
  id,
  nombre,
  descripcion,
  precio,
  duracionEstimada,
  activo,
  icono,
});
